src/models/addon.js

```diff
@@ -289,6 +285,24 @@ function parseAddonOptions (options) {
   }, {});
 }
 
+async function getPlan (planName, provider) {
+  // if no plan specified, pick the cheapest one
+  if (planName == null || planName === '') {
+    if (provider.plans.length === 0) {
+      throw new Error('no plans available');
```

Comment:
question: this will be displayed to the user, is it descriptive enough?

Reply:
This was not the subject of this PR, but I can do that.

Reply:
Do you have a more descriptive suggestion?

Reply:
I have introduced this if branch for code robustness. But, this should never happen if the API always returns plans for a available provider (which, as for now, is the case for every providers).

Reply:
Let's discuss it together in a meeting and move on :wink: